app/views/course/assessment/answer/forum_post_responses/_forum_post_response.json.jbuilder

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+json.fields do
+  json.questionId answer.question_id
+  json.id answer.acting_as.id
+  json.answer_text answer.answer_text
```

Comment:
Can we standardise? Either `camelCase` or `snake_case`. I would recommend going with `json.answerText`.

Reply:
Same thoughts on the standardisation, but other question types seem to have a blend of `camelCase` and `snake_case`. The former is typically used for `questionId`, and the latter for other variables specific to the question type.